<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RegistWindow</class>
 <widget class="QWidget" name="RegistWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>550</width>
    <height>550</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>551</width>
     <height>551</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <item>
     <widget class="QWidget" name="widget" native="true">
      <property name="styleSheet">
       <string notr="true">#widget {
background-image: url(:/images/timg.jpeg);
}</string>
      </property>
      <widget class="QFrame" name="frame">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>29</y>
         <width>431</width>
         <height>491</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">#frame {
	background-color: rgb(255, 255, 255);
	border-radius: 10px;
	background-image:none

}
QLabel {
	background-image: none;
}
QLineEdit {
    border-width: 2px;
    border-style: solid;
    border-radius: 10px;
	border-color:rgb(199, 199, 199);
	background-color:rgb(255, 255, 255);
}
</string>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <widget class="QPushButton" name="Regist">
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>420</y>
          <width>112</width>
          <height>32</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton{
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 rgba(227, 135, 193, 255), stop:1 rgba(251, 171, 182, 255));
color:rgb(255,255,255);
border-radius: 5px;
border:none;
}QPushButton::hover{
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:1 rgba(227, 135, 193, 255), stop:0 rgba(251, 171, 182, 255));
}
QPushButton::pressed{
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 rgba(227, 135, 193, 255), stop:1 rgba(251, 171, 182, 255));
}</string>
        </property>
        <property name="text">
         <string>注册</string>
        </property>
       </widget>
       <widget class="QPushButton" name="Reset">
        <property name="geometry">
         <rect>
          <x>250</x>
          <y>420</y>
          <width>112</width>
          <height>32</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton{
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 rgba(227, 135, 193, 255), stop:1 rgba(251, 171, 182, 255));
color:rgb(255,255,255);
border-radius: 5px;
border:none;
}QPushButton::hover{
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:1 rgba(227, 135, 193, 255), stop:0 rgba(251, 171, 182, 255));
}
QPushButton::pressed{
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 rgba(227, 135, 193, 255), stop:1 rgba(251, 171, 182, 255));
}</string>
        </property>
        <property name="text">
         <string>重置</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="UserLnEdit">
        <property name="geometry">
         <rect>
          <x>160</x>
          <y>40</y>
          <width>211</width>
          <height>41</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="EmailLnEdit">
        <property name="geometry">
         <rect>
          <x>160</x>
          <y>130</y>
          <width>211</width>
          <height>41</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="PwdLnEdit">
        <property name="geometry">
         <rect>
          <x>160</x>
          <y>220</y>
          <width>211</width>
          <height>41</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="ConfirmLnEdit">
        <property name="geometry">
         <rect>
          <x>160</x>
          <y>310</y>
          <width>211</width>
          <height>41</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label">
        <property name="geometry">
         <rect>
          <x>80</x>
          <y>40</y>
          <width>58</width>
          <height>41</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>18</pointsize>
         </font>
        </property>
        <property name="text">
         <string>用户名</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_2">
        <property name="geometry">
         <rect>
          <x>90</x>
          <y>220</y>
          <width>58</width>
          <height>41</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>18</pointsize>
         </font>
        </property>
        <property name="text">
         <string>密码</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_3">
        <property name="geometry">
         <rect>
          <x>60</x>
          <y>310</y>
          <width>81</width>
          <height>41</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>18</pointsize>
         </font>
        </property>
        <property name="text">
         <string>确认密码</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_4">
        <property name="geometry">
         <rect>
          <x>380</x>
          <y>230</y>
          <width>16</width>
          <height>21</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>18</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(255, 0, 0);</string>
        </property>
        <property name="text">
         <string>*</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_5">
        <property name="geometry">
         <rect>
          <x>380</x>
          <y>320</y>
          <width>16</width>
          <height>21</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>18</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(255, 0, 0);</string>
        </property>
        <property name="text">
         <string>*</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_6">
        <property name="geometry">
         <rect>
          <x>380</x>
          <y>50</y>
          <width>16</width>
          <height>21</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>18</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(255, 0, 0);</string>
        </property>
        <property name="text">
         <string>*</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_7">
        <property name="geometry">
         <rect>
          <x>90</x>
          <y>130</y>
          <width>58</width>
          <height>41</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>18</pointsize>
         </font>
        </property>
        <property name="text">
         <string>邮箱</string>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
